### PR TITLE
[WIP] Remove cname tracker lists

### PR DESCRIPTION
**Pull request by jellizaveta**
Related: 
https://github.com/AdguardTeam/FiltersCompiler/issues/217
https://github.com/AdguardTeam/FiltersRegistry/issues/946
Don't merge until this pr: https://github.com/AdguardTeam/FiltersRegistry/pull/953
It would be a better to optimise the CNAME entries together with the blocked servers file, so that the broader rules that already exist in adservers.txt and tracking_servers.txt can reduce the number of rules in the CNAME lists.

### CODE REVIEW

**Review by zloyden — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]

**Review by KatanaDi1408 — CHANGES_REQUESTED** [review]
****~~~~~~~~****

File: BaseFilter/sections/adservers.txt [hunk 1]
```diff
@@ -1105,3 +1105,257 @@
 !
 ! https://github.com/AdguardTeam/AdGuardHome/issues/1046
 ||samsungtvads.com^
+!
+! This section contains the list of the rules are against CNAME advertising networks.
+! We include rules from AdGuard's cname-tracker repo here, and
+! add some additional rules when we can block whole CNAME domain, not only CNAME subdomains.
```
Comment (Alex-302) [inline BaseFilter/sections/adservers.txt:1111]: > ...to optimize the filter size

or something like

> which will be used during the optimization by `/optimizeDomainBlockingRules` option.

Same below.

File: BaseFilter/sections/adservers.txt [hunk 2]
```diff
@@ -1105,3 +1105,257 @@
 !
 ! https://github.com/AdguardTeam/AdGuardHome/issues/1046
 ||samsungtvads.com^
+!
```
Comment (Alex-302) [inline BaseFilter/sections/adservers.txt:1108]: I'd wrap it by Comment Anchors tags

* Beginning
```
! NOTE: CNAME disguised advertising networks
! SECTION: CNAME disguised advertising networks
```

* End
```
! NOTE: CNAME disguised advertising networks end ⬆️
! !SECTION: CNAME disguised advertising networks
```

And same for tracking.

File: BaseFilter/sections/adservers.txt [hunk 3]
```diff
@@ -1105,3 +1105,257 @@
...
+! This section contains the list of the rules are against CNAME advertising networks.
```
Comment (Alex-302) [inline BaseFilter/sections/adservers.txt:1109]: Better `against CNAME` => `against CNAME disguised`... (and below)

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: BaseFilter/sections/adservers.txt [hunk 4]
```diff
@@ -1105,3 +1105,265 @@
...
+||advlistings.com^
+||minutelight-1.online^
+||revrtb.com^
+||newbluetrue.xyz^
+||onemediawork.com^
+!#include https://raw.githubusercontent.com/AdguardTeam/cname-trackers/master/data/ads/webair.txt
+!
+!
+! NOTE: CNAME disguised advertisin networks end ⬆️
+! !SECTION: CNAME disguised advertisin networks
```
Comment (Alex-302) [inline BaseFilter/sections/adservers.txt:1369]: a typo 
```suggestion
! !SECTION: CNAME disguised advertising networks
```

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**